variables.tf

```diff
@@ -58,9 +58,26 @@ variable ports {
   type        = "list"
 }
 
+variable health_check {
+  type = "map"
+  default = {
+    type = "tcp"
+    check_interval_sec = 5
+    healthy_threshold = 2
+    timeout_sec = 5
+    unhealthy_threshold = 2
+  }
+}
 variable http_health_check {
```

Comment:
This variable has been made obsolete by `health_check` so it should be removed. Additionally, we will need a migration guide which explains the change in the variable interface. terraform-google-project-factory and terraform-google-kubernetes-engine both have guides in their `docs` directories to reference.